opentracing-spring-web/src/main/java/io/opentracing/contrib/spring/web/interceptor/TracingHandlerInterceptor.java

```diff
@@ -94,6 +94,11 @@ public void afterConcurrentHandlingStarted (
             HttpServletRequest httpServletRequest, HttpServletResponse httpServletResponse, Object handler)
             throws Exception {
 
+        // exclude pattern, span is not started in filter
+        if (httpServletRequest.getAttribute(TracingFilter.SERVER_SPAN_CONTEXT) == null) {
```

Comment:
It's repeated in every method, could you please extract it in the separate method?